extension/app/manifest.json

```diff
@@ -1,41 +1,46 @@
 {
-	"name": "Interceptor",
-	"description": "Interceptor",
-	"version": "0.0.0",
-	"manifest_version": 2,
-	"browser_action": {
-		"default_popup": "html/popup.html",
-		"default_icon": {
-			"128": "img/head-not-active.png"
-		}
-	},
-	"background": {
-		"page": "html/background.html"
-	},
-	"icons": {
-		"128": "img/head.png",
-		"400": "img/LOGOA_400x400.png"
-	},
-	"permissions": [
-		"activeTab",
-		"<all_urls>",
-		"storage",
-		"webNavigation"
-	],
-	"web_accessible_resources": ["vendor/*", "js/*", "inpage/*"],
-	// all hashes except the first one are from .html files `<script type = 'module'>import '../js/popup.js'</script>` part
```

Comment:
I feel this is valuable commit. But if Firefox will approve interceptor after its, I think its worth getting rid of these. Is it sure that this is the solution, or do we try, and then bring these back if that was not the reason?